### ISSUE TITLE
npm install & node index.js is error...

### ISSUE THREAD

**Issue by Hiura-Souji** [SudhanPlayz/Discord-MusicBot#932]
npm install & node index.js is error...

**Include a reproducible code sample here, if possible:**

```js

D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma>npm install

> discord-musicbot@5.0.0-beta postinstall
> cd dashboard && curl --compressed -o- -L https://yarnpkg.com/install.sh | bash && yarn

'bash' Not an internal or external command, executable program or batch file.
npm ERR! code 255
npm ERR! path D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c cd dashboard && curl --compressed -o- -L https://yarnpkg.com/install.sh | bash && yarn

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Administrator\AppData\Local\npm-cache\_logs\2022-06-22T05_24_55_346Z-debug-0.log

D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma>node index.js
Make sure to fill in the config.js before starting the bot.
[22:5:2022 - 13:25] | Started the bot...
D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma\lib\DiscordMusicBot.js:50
        throw Error(err);
              ^

Error: TypeError: Cannot read properties of undefined (reading 'startsWith')
    at D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma\lib\DiscordMusicBot.js:50:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.4.0
```

**Further details:**

- discord.js version:
- Node.js version: 18.4.0
- Operating system:
- Priority this issue should have – please be realistic and elaborate if possible:

**Relevant client options:**

- partials: none
- gateway intents: none
- other: none

**Comment by JotaroKujo0525** [comment]
First, bash doesn't work in windows, so remove the `postinstall` array in `package.json`.
Second,this could've been a package fault, so try deleting `node_modules`

tldr:
Remove `postinstall ` array on package.json, then npm i again. And now `node index.js`

**Comment by JotaroKujo0525** [comment]
This isn't a bug, but just a fault in code. If this solves your issue, you can now close this issue

**Comment by SudhanPlayz** [comment]
fixed

**Comment by JotaroKujo0525** [comment]
h

**Comment by Hiura-Souji** [comment]
hello
thank you for your help
now the msg is show:

`D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma>node index.js
Make sure to fill in the config.js before starting the bot.
[22:5:2022 - 15:15] | Started the bot...
D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma\lib\DiscordMusicBot.js:50
        throw Error(err);
              ^

Error: TypeError: Cannot read properties of undefined (reading 'startsWith')
    at D:\_dc_bot\Music\Discord-MusicBot\v5_Aruma\lib\DiscordMusicBot.js:50:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.4.0`

**Comment by JotaroKujo0525** [comment]
Must've been your failed installation earlier

**Comment by Hiura-Souji** [comment]
i am so sorry
it is config have a new val
and i use old ver config......

it is can run now.
thank you~
